camayoc/tests/qcs/cli/test_scans.py

```diff
@@ -0,0 +1,290 @@
+# coding=utf-8
+"""Tests for ``qpc scan`` commands.
+
+:caseautomation: automated
+:casecomponent: cli
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+import operator
+import re
+import time
+from pprint import pformat
+
+import pexpect
+import pytest
+
+from .conftest import qpc_server_config
+from .utils import (
+    cred_add,
+    source_add,
+    scan_cancel,
+    scan_pause,
+    scan_restart,
+    scan_show,
+    scan_start,
+)
+from camayoc.config import get_config
+from camayoc.constants import BECOME_PASSWORD_INPUT, CONNECTION_PASSWORD_INPUT
+from camayoc.exceptions import (
+    ConfigFileNotFoundError,
+    FailedScanException,
+    WaitTimeError,
+)
+
+
+name_getter = operator.itemgetter('name')
+"""Fetche the value of the ``name`` item."""
```

Comment:
`s/Fetche/Fetch/` 

Now that I understand more what is going on here, this is what I would say:
```
name_getter = operator.itemgetter('name')
"""Fetch the value of the ``name`` item.

Function handle that returns the value associated with the 'name' key of any
dictionary like object passed to it. 
See https://docs.python.org/3.5/library/operator.html#operator.itemgetter

Used to generate test id's.
"""
```

I understand there is an aspect of RTFD as far as I can go look up those docs to find out it is a function handle and accepts dict like objects but that is a bit deeper than I'd like to have to dig.

Reply:
I see comments in a different way, I would let the code say as much as it can, for example "Fetch the value of the ``name`` item" is already said by the code itself. Also referencing std lib and paraphrasing it is something that I tend to avoid because any information about the language can be checked there.

I think comments are useful to express some intentions specially when some fancy logic is used.  I have failed on finding a good comment for this but now with your suggestion I think I a good one would be "Generate test IDs by fetching the ``name`` item's value"

Reply:
If you continue feeling uncomfortable by not having more detailed information let me know that I can add that later.